### ISSUE TITLE
plugin libraries fail to load

### ISSUE THREAD

**Issue by crowell** [Vector35/binaryninja-api#285]
```
minishwoods binary_ninja/binaryninja » ./binaryninja /bin/ls                                                 1 ↵

(binaryninja:27870): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(binaryninja:27870): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(binaryninja:27870): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Plugin module '/home/jeff/ctf/tools/binary_ninja/binaryninja/plugins/libcryptoplugin.so' failed to load
Plugin module '/home/jeff/ctf/tools/binary_ninja/binaryninja/plugins/libssl.so' failed to load
IL for flag c in instruction add.q{*}(rax, [rdi + 0x30].q) at 0x412610 could not be generated
IL for flag c in instruction add.d{*}(ebx, 1) at 0x40c2ae could not be generated
```

on debian sid

**Comment by psifertex** [comment]
SID is officially unsupported, but I'll give it a whirl and see if I anythinf jumps out.

**Comment by psifertex** [comment]
This is a bug in our build process that is masked in ubuntu because they happen to have libraries with the right name. Temporary fix until we fix the build server is to rename `libssl.so` to `libssl.so.1.0.0` and likewise with `libcryptoplugin.so` to `libcryptoplugin.so.1.0.0` in the `plugins/` folder.

**Comment by cbisnett** [comment]
I can confirm this issue also affects Fedora 23. Renaming `libssl.so` to `libssl.so.1.0.0` and `libcrypto.so` to `libcrypto.so.1.0.0` also fixed the issue for me

This is how I confirmed the issue:
```
[chris@xps13 binary_ninja]$ ldd plugins/libcryptoplugin.so 
	linux-vdso.so.1 (0x00007ffeddbe9000)
	libbinaryninjacore.so.1 => not found
	libcrypto.so.1.0.0 => not found
	libssl.so.1.0.0 => not found
	libstdc++.so.6 => /lib64/libstdc++.so.6 (0x00007f78f68ac000)
	libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f78f6694000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f78f62d3000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f78f5fd1000)
	/lib64/ld-linux-x86-64.so.2 (0x0000557148d66000)
```
libbinaryninjacore.so.1 was not found by `ldd` but is in the Binary Ninja root directory.

**Comment by plafosse** [comment]
We've recently improved support for non ubuntu linux boxes. This should be fixed in 1.1.1009. Please reopen if this isn't fixed.
